express-notes-api/server.js

```diff
@@ -0,0 +1,109 @@
+import express from 'express';
+
+import { readFile, writeFile } from 'node:fs/promises';
+
+const app = express();
+const json = JSON.parse(await readFile('data.json'));
+let message = {};
```

Comment:
This shouldn't be a global variable. Just declare it inside each function right before you set it.